README.md

```diff
@@ -12,6 +12,95 @@ If you create a connected app using below link, you need not pass client Id and
 
 https://extensiongallery.tableau.com/connectors/270
 
+Alternatively, if you want to use key pair authentication instead of username and password, please follow the below steps to generate a key pair and configure public certificate with your connected app.
+
+### Create a Private Public Key Pair and a Public Digital Certificate 
+Open terminal and change directories to any folder and run the following commands:
+
+#### 1. Create a private public key pair
+```
+openssl genrsa -out keypair.key 2048
+```
+#### 2. Create a digital certificate from the key pair
+```
+openssl req -new -x509 -nodes -sha256 -days 365 -key keypair.key -out certificate.crt
+```
+> **Note:** There will be a series of questions (Country, State, Org Name, etc) you need to complete.
+#### 3. Create pkcs8 private key from the key pair
+```
+openssl pkcs8 -topk8 -nocrypt -in keypair.key -out private.key
+```
+
+![Terminal key generation screenshot](images/terminal-key-generation.png)
+> This creates keypair.key, private.key and certificate.crt files in that folder. Files private.key and certificate.crt will be used later in the setup.
+### Connected App Setup
+
+This flow utilizes Salesforce's server to server [JWT bearer flow](https://help.salesforce.com/articleView?id=remoteaccess_oauth_jwt_flow.htm&type=5) for acquiring an access token. This portion of the setup will walk you through setting up the connected app.
+
+1. Login to Salesforce → Setup and Search "**App Manager**"
+2. In the Setup’s Quick Find search "**App Manager**"
+3. Select **“New Connected App”**
+    1. Connected App Name: CDP API
+    2. API Name: CDP_API (or whatever default value is pre-populated)
+    3. Contact Email: Your email address
+    4. Under API Heading, check the box for **“Enable OAuth Settings”**
+    5. Callback URL: https://oauth.pstmn.io/v1/callback
```

Comment:
This is postman call back URL. Customer can provide his own callback URL too. Good to mention it or provide salesforce public doc link for creation of connected app

Reply:
Agreed.